content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -323,6 +323,14 @@ kubectl exec my-pod -c my-container -- ls /         # Run command in existing po
 kubectl top pod POD_NAME --containers               # Show metrics for a given pod and its containers
 kubectl top pod POD_NAME --sort-by=cpu              # Show metrics for a given pod and sort it by 'cpu' or 'memory'
 ```
+## Copy files and directories to and from containers
+
+```bash
+kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # Copy /tmp/foo_dir local directory to /tmp/bar_dir in a remote pod in the default namespace
```

Comment:
```suggestion
kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # Copy /tmp/foo_dir local directory to /tmp/bar_dir in a remote pod in the current namespace
```
I would mention the limitations (eg: this typically wouldn't work for a Pod on a Windows node, because it execs and tries to run `tar`).

Reply:
I have made the suggested changes.